### PR TITLE
ICMSLST-1206 Add Search actions for Import applications.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/templates/web/domains/case/search/import/actions.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ul class="menu-out">
+  {% for action in rec.actions  %}
+    <li>
+      <form method="post" action="{{ action.url }}">
+        <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}"/>
```
Comment (davidwtbuxton) [inline web/templates/web/domains/case/search/import/actions.html:5]: Worth using `{{ csrf_input }}` here? https://docs.djangoproject.com/en/3.2/ref/csrf/#using-csrf-in-jinja2-templates

**Review by davidwtbuxton — APPROVED** [review]

File: web/templates/web/domains/case/search/import/actions.html (hunk 1 above)
Comment (MattHolmes123) [reply web/templates/web/domains/case/search/import/actions.html:5]: yes my bad.
I copied the workbasket template

Comment (MattHolmes123) [reply web/templates/web/domains/case/search/import/actions.html:5]: Thanks

File: web/templates/web/domains/case/search/import/actions.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <button type="submit" class="link-button {{ action.icon }} button">
+          {{ action.link_text }}
```
Comment (MattHolmes123) [inline web/templates/web/domains/case/search/import/actions.html:7]: Bug from a refactor.
Should be {{ action.name }}

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**